packages/admin/cms-admin/src/dependencies/DependencyList.tsx

```diff
@@ -112,14 +112,14 @@ export const DependencyList = ({ query, variables }: DependencyListProps) => {
                     return <FormattedMessage id="comet.dependencies.dataGrid.cannotLoadUrl" defaultMessage="Cannot determine URL" />;
                 }
 
-                const loadUrl = () => {
-                    return dependencyObject.resolveUrl({
+                const loadUrl = async () => {
+                    const path = await dependencyObject.resolveUrl({
                         rootColumnName: row.rootColumnName,
                         jsonPath: row.jsonPath,
-                        contentScopeUrl: contentScope.match.url,
                         apolloClient,
                         id: row.id,
                     });
+                    return contentScope.match.url + path;
```

Comment:
I don't think we'll support dependencies across content scopes in the foreseeable future (and it won't work `contentScope.match.url` anyway), so I just append it here directly.

Reply:
Fine by me